app/controllers/api/v2/provisioning_templates_controller.rb

```diff
@@ -0,0 +1,133 @@
+module Api
+  module V2
```

Comment:
+1 for keeping the seprata controller and views,  for the reasons @ares mentioned. The DRYing doesn't help here much, as the old code should not be touched by new changes: new attributes should not be added to old version etc.